Src/LogData.c

```diff
@@ -45,18 +52,19 @@ typedef struct{
     int32_t altitude;
     int32_t currentFlightPhase;
     int32_t tick;
-} logEntry; // logEntry holds data from AllData that is to be logged
+} LogEntry; // logEntry holds data from AllData that is to be logged
 
 
 /* Constants -----------------------------------------------------------------*/
-static int SLOW_LOG_DATA_PERIOD = 700;
-static int FAST_LOG_DATA_PERIOD = 200;
-static uint8_t softwareVersion = 104;
-static uint8_t deviceAddress = 0x50;
-static uint8_t currMemAddress = 0x07; //TODO: update this as needed
-static uint32_t timeout = 10; //Time required to wait before ending read/write (unit =?)
-static logEntry theLogEntry; //This is the global log entry that is updated
-static uint8_t logEntrySize = 84;
+static const int32_t SLOW_LOG_DATA_PERIOD_ms = 700; // logging period for slow routine
+static const int32_t FAST_LOG_DATA_PERIOD_ms = 200; // logging period for fast routine
+static const uint8_t softwareVersion = 104; // this is not used at all
+static const uint8_t deviceAddress = 0x50; // used for reading/writing to EEPROM
+static uint8_t EEPROM_START_ADDRESS = 0x07; // start address for reading/writing
+static uint8_t logAddressOffset = 0; // offset updated after writing in logEntryOnceRoutine
+static const uint32_t timeout_ms = 10; // Time required to wait before ending read/write
+static LogEntry theLogEntry; // This is the global log entry that is updated
+static const uint8_t LogEntrySize = sizeof(LogEntry); //size of logEntry = 96 bytes
```

Comment:
I would actually just use `sizeof(LogEntry)` instead of having the `LogEntrySize` constant at all. For example on line 141: `char dataRead[sizeof(LogEntry)];`